examples/ads.amp.html

```diff
@@ -195,20 +195,7 @@ <h2 id="amoad" class="broken">AMoAd</h2>
       data-sid="62056d310111552c951d19c06bfa71e16e615e39493eceff667912488bc576a6">
   </amp-ad>
 
-  <h2 id="appnexus" class="broken">AppNexus single ad with tagid (placement id)</h2>
```

Comment:
Can you track down who contributed the appnexus ad and file a bug against them to add a valid example so we make sure we don't break their ads.